Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Stage: Composer install for production
+#
+FROM composer:1.7.3 AS composer
+
+COPY composer.json \
+    composer.lock \
+    symfony.lock \
+    ./
+
+RUN composer --no-interaction install --no-dev --ignore-platform-reqs --no-autoloader --no-suggest --prefer-dist
+
+COPY src/ src/
+
+RUN composer --no-interaction dump-autoload --classmap-authoritative
+
+
+
+#
+# Stage: Composer install for development
+#
+FROM composer AS composer-dev
+
+RUN composer --no-interaction install --ignore-platform-reqs --no-suggest --prefer-dist
+
+
+
+#
+# Stage: Production environment
```

Comment:
Maybe should set `APP_ENV` in these? 🤔

Reply:
Two points:

- could be accidentally inherited into the wrong target, but all overrides will happen here so easy to check
- if it's actually hardcoded into a single target, why do all the targets have all the configuration overrides like `config/packages/*` subfolders? (because it's probably simpler) But they could be resolved at image build time too.

Reply:
It could be overridden at runtime. Would be weird to mix them.

Don't think it's possible to ignore subdirectories in a `COPY`?

Reply:
No, would be science fiction, if the Symfony standard is that one we should ship the folder as-is